ush/python/pyobsforge/task/marine_bufr_prepobs.py

```diff
@@ -14,14 +19,23 @@ class MarineBufrObsPrep(Task):
     def __init__(self, config: Dict[str, Any]) -> None:
         super().__init__(config)
 
+        # TODO: for development purposes, remove when dumpdir and all the cycles line up
+        self.task_config.current_cycle = datetime(2025, 4, 6, 12, 0, 0)
+        self.task_config.RUN = "gdas"
+        self.task_config.PDY = self.task_config.current_cycle.strftime("%Y%m%d")
+        self.task_config.cyc = self.task_config.current_cycle.strftime("%H")
+
         _window_begin = add_to_datetime(self.task_config.current_cycle, -to_timedelta(f"{self.task_config['assim_freq']}H") / 2)
         _window_end = add_to_datetime(self.task_config.current_cycle, +to_timedelta(f"{self.task_config['assim_freq']}H") / 2)
 
         local_dict = AttrDict(
             {
+                'COMOUT': f"{self.task_config.HOMEobsforge}/COMROOT/obsforge/gfs.20250428/18/ocean/",
                 'window_begin': _window_begin,
                 'window_end': _window_end,
-                'PREFIX': f"{self.task_config.RUN}.t{self.task_config.cyc:02d}z.",
+                'PREFIX': f"{self.task_config.RUN}.t{self.task_config.cyc}z.",
+                'bufr2ioda_config_temp': f"{self.task_config.HOMEobsforge}/parm/{self.task_config.BUFR2IODA_CONFIG_TEMP}",
+                'DMPDIR': f"{self.task_config.DMPDIR_BUFR}/{self.task_config.RUN}.{self.task_config.PDY}/{self.task_config.cyc}/atmos"
```

Comment:
Take a look at the workflow, they've changed some names that make things clearer like COMIN_OBSPROC or something like that